demo/lightning/checkpoint/singlenode/continuous.py

```diff
@@ -69,7 +69,11 @@ def main():
                       accelerator="cpu",
                       strategy="ddp",
                       num_nodes=1)
+    t1 = time.time()
     trainer.fit(model, dataloader)
+    t2 = time.time()
+    result_time = t2 - t1
+    print(f"Total elapsed time was {result_time} seconds...")
```

Comment:
nit: Because the call to trainer.fit includes all the time it takes to initialize and do a step on the model, if you want real timing info it's better to use the checkpoint benchmark code instead of the demo code